Resources/config/services.yml

```diff
@@ -1,4 +1,5 @@
 parameters:
+    ezsystems.platformui.twig.text_extension.class: Twig_Extensions_Extension_Text
```

Comment:
@adamwojs I guess I should skip this as well here, right?

Reply:
Yes, I've enabled this Twig extension because we need `truncate` filter for Link Manager.

Reply:
Thanks, omitted it then.